src/reducer.js

```diff
@@ -468,6 +470,11 @@ const onSetWallet = (state, action) => {
   };
 };
 
+const onSetUseWalletService = (state, action) => ({
+  ...state,
+  useWalletService: action.payload,
```

Comment:
We are not using this state anywhere in the application yet, right?

Reply:
We are using it, it is loaded from the props on the following screens:

```
src/screens/CreateTokenConfirm.js
src/screens/SendConfirmScreen.js
src/screens/LoadHistoryScreen.js
```